ScoreSaber/UI/Leaderboard/ScoreSaberLeaderboardViewController.cs

```diff
@@ -274,68 +458,69 @@ public async Task RefreshLeaderboard(BeatmapLevel beatmapLevel, BeatmapKey beatm
                     return;
                 }
 
+                if(!beatmapKey.levelId.Contains("custom_level_")) {
```

Comment:
same thing as always

Reply:
beatmapkey doesnt expose it

Reply:
But you have the beatmapLevel available…

Reply:
sorry, got it mixed up, fixed